app/assets/images/payment_icons/bangkokbank.svg

```diff
@@ -0,0 +1 @@
0-.2 0-.2.3v20h-.1V22 2c0-.1 0-.2.2-.2a15210 15210 0 0 0 20.4.2v20.2"/><path fill="#F9FAFB" d="M9 21.9v-.2V2.4C9 2 9 2 9.3 2h19.5c.2 0 .2 0 .2.3v19.5H29v-.3V2.5c0-.3 0-.3-.2-.3H9.4c-.3 0-.3 0-.3.3v19.3"/><path fill="#F9FAFB" d="M28.8 22H9.4h-.2v-.2h.2a30812.9 30812.9 0 0 1 19.4 0v.1"/><path fill="#CFD3E6" d="M9.2 21.8v.1S9 22 9 22h.2zM28.8 22v-.2h.2c0 .2-.1.1-.2.1"/><path fill="#1B2B88" d="M28.9 21.8h-.3a30812.9 30812.9 0 0 0-19.4 0v-.2V2.5c0-.3 0-.3.2-.3h19.3c.2 0 .2 0 .2.3v19.3m-5.4-10.3c0-.7-.2-1.4-.6-2.1L21.3 7l-2.2-2.9c-.2-.3-.2-.3-.4 0l-2.4 3.1-1.6 2.4c-.2.4-.4 1-.4 1.5v1c0 .4.2.8.5 1.2a52.5 52.5 0 0 0 4.3 3.1l1-.7c.8-.6 1.8-1.2 2.6-2 .3-.3.6-.7.7-1.2l.1-1.1m0 7.4c.2.4.4.5.8.4h.5c.2 0 .2 0 .2-.2l-.1-.3v-.5c0-.2 0-.3-.2-.3-.4-.2-.7-.1-1 0v.4h.3c.1 0 0-.1 0-.2h.3v.2c-.3.1-.6.1-.8.5 0-.3 0-.5-.3-.6.2-.1.3-.3.3-.5s-.2-.4-.5-.4h-1c-.2 0-.2.1-.2.2l.1.3v1.2c-.1.1 0 .2 0 .2h1c.3 0 .5 0 .6-.4m-12-.6c.2-.1.3-.3.3-.5s-.2-.4-.5-.4a62 62 0 0 0-1.2.2l.1.4V19c-.1.1 0 .2 0 .2h1c.2 0 .4 0 .5-.2.2-.3.1-.6-.2-.8m3.5.3v.2c0 .5.4.6.7.5 0 0 .1-.1.2 0l-.1.3h-.2c0-.3-.2-.2-.3-.2H15l.1.3.7.2c.3 0 .5-.2.5-.6v-1-.2l.1-.1h-.2l-.8-.1c-.3 0-.5.1-.5.4v.3m12.3.2l.2.2c.1.3.3.3.5.3h.2V19l-.4-.5v-.3l.2-.1c.1 0 .2-.1.1-.2 0-.1 0 0-.1 0h-.2s-.3-.1-.2.1l-.1.1-.1.1c-.1 0 0 0 0-.1v-.4c0-.4 0-.4-.5-.4-.2 0-.2.1-.2.2l.1.4a57.4 57.4 0 0 1 0 1.3h.4c.2 0 .2 0 .2-.2v-.3m-7.3 0l.3.3c0 .2.4.3.5.2.1 0 .1-.1 0-.2l-.2-.4c-.2-.2-.2-.2 0-.5.1 0 .3 0 .2-.2h-.3l-.3.1v.2H20v-.5c0-.4 0-.4-.5-.4-.1 0-.2.1 0 .2v1.5c-.2.1-.1.2 0 .2h.4c.2 0 .3 0 .2-.2v-.3m-3-.5v-.5c0-.4 0-.4-.5-.4-.2 0-.2.2-.2.2.2.1.1.3.1.4v1.1c-.1.1 0 .2 0 .2h.5c.2 0 .2 0 .1-.2v-.3l.1.2c.1.3.3.3.6.3h.1V19l-.4-.5v-.2l.3-.3s.1 0 0-.1h-.5c0 .2-.1.3-.3.3m-2.3.3v-.3c0-.3-.2-.5-.4-.4h-.8c-.2 0-.3.1-.2.3.2.3.2.6 0 .9l.1.2h.5l.1-.1c-.1-.3 0-.5 0-.8 0 0-.1-.2 0-.2s.2 0 .2.2v.4c0 .6 0 .5.5.5.1 0 .2-.1 0-.2v-.5m3.2 0v.3c0 .2.1.3.2.3.3.2.6.2 1 0 .1 0 .2 0 .2-.3v-.5c0-.2 0-.4-.2-.5h-.7c-.2 0-.4 0-.4.3-.1.1 0 .3 0 .4m7.9.2c0 .6 0 .5.4.5.2 0 .3 0 .2-.2l-.1-.3v-.5c0-.3-.2-.5-.4-.4h-.8c-.2 0-.2.1-.2.3.2.3.2.6 0 1l.1.1h.5s.2 0 .1-.1c-.1-.3 0-.6 0-.8v-.2c.1 0 .2 0 .2.2v.4m-12.8-.1v-.4c0-.2 0-.3-.3-.4l-.9.1v.3l.1.1c.2 0 .3 0 .3-.2H12.7l-.1.2-.4.1c-.2.1-.4.3-.3.5 0 .2.2.3.4.3h.9c.1 0 .1 0 0-.1v-.5"/><path fill="#1B2B88" d="M21.5 9.5c0 .4-.1.6-.3.9L19 13h-.2l-1.4-1.7-.7-.9c-.3-.2-.4-.6-.4-1a95.8 95.8 0 0 1 2.4-3.6c.2-.2.2-.2.4 0L20.6 8l.7 1 .2.6M18.1 14.9l-.8-.6-1.2-.9a2 2 0 0 1-.7-.9c-.2-.6-.2-1.2 0-1.8l.1-.3.2.2 2.3 2.9c.2.3.3.6.2 1v.4M19.6 14.3c0-.3 0-.6.2-.8l.7-.8 1.6-2.2s0-.1.1 0h.1c.2.5.3 1 .2 1.5 0 .6-.4 1-.9 1.5l-1.8 1.3s-.1.1-.2 0v-.5M22.4 18.7c0-.1 0-.2.2-.2s.3 0 .3.2c0 .1 0 .2-.3.2-.2 0-.2 0-.2-.2M22.6 18.2c-.1 0-.2 0-.2-.2 0-.1 0-.2.2-.2s.3 0 .3.2-.1.2-.3.2M24.4 18.8c0 .2 0 .2-.2.2l-.1-.1.2-.2.1.1M10.8 18.7l.1-.2c.2 0 .3 0 .3.2s-.1.2-.3.2c-.2 0-.1-.1-.1-.2M10.8 18l.1-.2c.2 0 .3 0 .3.2 0 .1-.1.2-.3.2l-.1-.2M15.9 18.6v.2l-.1.2-.3-.1v-.6h.3v.3M18.6 18.6v-.3c0-.1 0-.2.1-.1l.2.1v.6c0 .1 0 .2-.2.2l-.1-.2v-.3zM12.7 18.8l-.1.2-.2-.1.2-.2.1.1"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="38" height="24" viewBox="0 0 38 24" role="img" aria-labelledby="pi-bangkokbank"><title id="pi-bangkokbank">Bangkok Bank</title><path opacity=".1" d="M35 0H3a3 3 0 0 0-3 3v18a3 3 0 0 0 3 3h32a3 3 0 0 0 3-3V3a3 3 0 0 0-3-3z"/><path fill="#FFF" d="M35 1a2 2 0 0 1 2 2v18a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V3c0-1.1.9-2 2-2h32"/><path fill="#1B2B88" d="M29.2 22.2H8.8v-.4V2.2c0-.3 0-.3.2-.3h20c.2 0 .2 0 .2.3v20m-.4-.3c.1 0 .2.1.2 0v-.2V2.3c0-.2 0-.2-.2-.2H9.3c-.3 0-.3 0-.3.3v19.5c0 .1.2 0 .2 0H28.8"/><path fill="#F9FAFB" d="M29.2 22.2v-.3V2.2c0-.4 0-.3-.2-.3H9c-.2 0-.2 0-.2.3v20h-.1V22 2c0-.1 0-.2.2-.2a15210 15210 0 0 0 20.4.2v20.2"/><path fill="#F9FAFB" d="M9 21.9v-.2V2.4C9 2 9 2 9.3 2h19.5c.2 0 .2 0 .2.3v19.5H29v-.3V2.5c0-.3 0-.3-.2-.3H9.4c-.3 0-.3 0-.3.3v19.3"/><path fill="#F9FAFB" d="M28.8 22H9.4h-.2v-.2h.2a30812.9 30812.9 0 0 1 19.4 0v.1"/><path fill="#CFD3E6" d="M9.2 21.8v.1S9 22 9 22h.2zM28.8 22v-.2h.2c0 .2-.1.1-.2.1"/><path fill="#1B2B88" d="M28.9 21.8h-.3a30812.9 30812.9 0 0 0-19.4 0v-.2V2.5c0-.3 0-.3.2-.3h19.3c.2 0 .2 0 .2.3v19.3m-5.4-10.3c0-.7-.2-1.4-.6-2.1L21.3 7l-2.2-2.9c-.2-.3-.2-.3-.4 0l-2.4 3.1-1.6 2.4c-.2.4-.4 1-.4 1.5v1c0 .4.2.8.5 1.2a52.5 52.5 0 0 0 4.3 3.1l1-.7c.8-.6 1.8-1.2 2.6-2 .3-.3.6-.7.7-1.2l.1-1.1m0 7.4c.2.4.4.5.8.4h.5c.2 0 .2 0 .2-.2l-.1-.3v-.5c0-.2 0-.3-.2-.3-.4-.2-.7-.1-1 0v.4h.3c.1 0 0-.1 0-.2h.3v.2c-.3.1-.6.1-.8.5 0-.3 0-.5-.3-.6.2-.1.3-.3.3-.5s-.2-.4-.5-.4h-1c-.2 0-.2.1-.2.2l.1.3v1.2c-.1.1 0 .2 0 .2h1c.3 0 .5 0 .6-.4m-12-.6c.2-.1.3-.3.3-.5s-.2-.4-.5-.4a62 62 0 0 0-1.2.2l.1.4V19c-.1.1 0 .2 0 .2h1c.2 0 .4 0 .5-.2.2-.3.1-.6-.2-.8m3.5.3v.2c0 .5.4.6.7.5 0 0 .1-.1.2 0l-.1.3h-.2c0-.3-.2-.2-.3-.2H15l.1.3.7.2c.3 0 .5-.2.5-.6v-1-.2l.1-.1h-.2l-.8-.1c-.3 0-.5.1-.5.4v.3m12.3.2l.2.2c.1.3.3.3.5.3h.2V19l-.4-.5v-.3l.2-.1c.1 0 .2-.1.1-.2 0-.1 0 0-.1 0h-.2s-.3-.1-.2.1l-.1.1-.1.1c-.1 0 0 0 0-.1v-.4c0-.4 0-.4-.5-.4-.2 0-.2.1-.2.2l.1.4a57.4 57.4 0 0 1 0 1.3h.4c.2 0 .2 0 .2-.2v-.3m-7.3 0l.3.3c0 .2.4.3.5.2.1 0 .1-.1 0-.2l-.2-.4c-.2-.2-.2-.2 0-.5.1 0 .3 0 .2-.2h-.3l-.3.1v.2H20v-.5c0-.4 0-.4-.5-.4-.1 0-.2.1 0 .2v1.5c-.2.1-.1.2 0 .2h.4c.2 0 .3 0 .2-.2v-.3m-3-.5v-.5c0-.4 0-.4-.5-.4-.2 0-.2.2-.2.2.2.1.1.3.1.4v1.1c-.1.1 0 .2 0 .2h.5c.2 0 .2 0 .1-.2v-.3l.1.2c.1.3.3.3.6.3h.1V19l-.4-.5v-.2l.3-.3s.1 0 0-.1h-.5c0 .2-.1.3-.3.3m-2.3.3v-.3c0-.3-.2-.5-.4-.4h-.8c-.2 0-.3.1-.2.3.2.3.2.6 0 .9l.1.2h.5l.1-.1c-.1-.3 0-.5 0-.8 0 0-.1-.2 0-.2s.2 0 .2.2v.4c0 .6 0 .5.5.5.1 0 .2-.1 0-.2v-.5m3.2 0v.3c0 .2.1.3.2.3.3.2.6.2 1 0 .1 0 .2 0 .2-.3v-.5c0-.2 0-.4-.2-.5h-.7c-.2 0-.4 0-.4.3-.1.1 0 .3 0 .4m7.9.2c0 .6 0 .5.4.5.2 0 .3 0 .2-.2l-.1-.3v-.5c0-.3-.2-.5-.4-.4h-.8c-.2 0-.2.1-.2.3.2.3.2.6 0 1l.1.1h.5s.2 0 .1-.1c-.1-.3 0-.6 0-.8v-.2c.1 0 .2 0 .2.2v.4m-12.8-.1v-.4c0-.2 0-.3-.3-.4l-.9.1v.3l.1.1c.2 0 .3 0 .3-.2H12.7l-.1.2-.4.1c-.2.1-.4.3-.3.5 0 .2.2.3.4.3h.9c.1 0 .1 0 0-.1v-.5"/><path fill="#1B2B88" d="M21.5 9.5c0 .4-.1.6-.3.9L19 13h-.2l-1.4-1.7-.7-.9c-.3-.2-.4-.6-.4-1a95.8 95.8 0 0 1 2.4-3.6c.2-.2.2-.2.4 0L20.6 8l.7 1 .2.6M18.1 14.9l-.8-.6-1.2-.9a2 2 0 0 1-.7-.9c-.2-.6-.2-1.2 0-1.8l.1-.3.2.2 2.3 2.9c.2.3.3.6.2 1v.4M19.6 14.3c0-.3 0-.6.2-.8l.7-.8 1.6-2.2s0-.1.1 0h.1c.2.5.3 1 .2 1.5 0 .6-.4 1-.9 1.5l-1.8 1.3s-.1.1-.2 0v-.5M22.4 18.7c0-.1 0-.2.2-.2s.3 0 .3.2c0 .1 0 .2-.3.2-.2 0-.2 0-.2-.2M22.6 18.2c-.1 0-.2 0-.2-.2 0-.1 0-.2.2-.2s.3 0 .3.2-.1.2-.3.2M24.4 18.8c0 .2 0 .2-.2.2l-.1-.1.2-.2.1.1M10.8 18.7l.1-.2c.2 0 .3 0 .3.2s-.1.2-.3.2c-.2 0-.1-.1-.1-.2M10.8 18l.1-.2c.2 0 .3 0 .3.2 0 .1-.1.2-.3.2l-.1-.2M15.9 18.6v.2l-.1.2-.3-.1v-.6h.3v.3M18.6 18.6v-.3c0-.1 0-.2.1-.1l.2.1v.6c0 .1 0 .2-.2.2l-.1-.2v-.3zM12.7 18.8l-.1.2-.2-.1.2-.2.1.1"/></svg>
```

There's an extra chevron on `<title>`; currently it says `<title id="pi-bangkokbank">Bangkok Bank</title>>`. This suggestion fixes that.